policies/late_night.md

```diff
@@ -0,0 +1,32 @@
+# Late Night Policy
+
+Last Updated: **NOT CURRENTLY RATIFIED  POLICY OF FARSET LABS**
+
+Next Review: **NOT CURRENTLY RATIFIED  POLICY OF FARSET LABS**
+
+## Late Events
+
+All events or gatherings which involve alcohol or loud music must end by 10.30 and vacate the building by 11pm. 
+
+Occasional members only gatherings may exceed this time but any loud music/PA must be disabled by 10.30
+
+## Late Guests
+
+**Non-member social guests are not permitted on the premises after 11pm.** If members are working collaboratively with non-members into the night that is acceptable. Members working collaboratively late at night with non-members must take responsibility for the non-members,
+including ensuring the number of non-members does not exceed the number of members working and alcohol consumption is kept to a minimum. This applies at all times but is particularly important at night.
+
+**Farset Labs should NEVER be an option for drinks after bars are closed.** Groups of drunk people arriving at our studio after chucking out time is not appropriate in a residential area and jeopardises the good will within the community for everyone. Absolutely NO exceptions.
+
+By all means have a few drinks if you're working late, but outside community run events, it's not OK to just hang out in the space areas drinking.
+
+## Late Working
+
+Farset Labs is accessible to members 24/7 **at the pleasure of Weavers Court Business Park**. 
+
+While you may stay in the space till the small hours of the morning, do not expect to be let in to the park complex automatically in the middle of the night.
+
+## Sleeping
+
+**Sleeping is not permitted in the space, regardless of time of day.** While there is often a temptation to just put the head down to work overnight, "You don't have to go home but you can't stay here.
+
+There *may* be exceptions made to this rule for some specific events such as hackathons etc, but those exceptions will be actively signposted ahead of the event.
```

Comment:
Add something to the effect of "**Never** assume that it's okay to sleep in the space."